restful-ws/3.1/_index.md

```diff
@@ -28,20 +28,20 @@ may be found on the [Project Site](https://projects.eclipse.org/projects/ee4j.ja
 
 ## Plan Review
 
-The Specification Committee Ballot concluded successfully on 2021-04-29 with the following results.
```

Comment:
The plan review ballot results must be kept.
A release review section will be added after the release review has completed

Reply:
Restored.